pages/treeherder.py

```diff
@@ -91,15 +125,27 @@ def unclassified_failure_count(self):
     def clear_filter(self):
         self.selenium.find_element(*self._clear_filter_locator).click()
 
+    def clear_filter_by_shortcut(self):
+        el = self.find_element(*self._result_sets_locator)
+        el.send_keys(Keys.CONTROL + Keys.SHIFT + 'f')
```

Comment:
I would send keys to the `<body>` as that's more representative of what the user is likely to be doing.